### DISCUSSION TITLE
Rdio Scanner app and the website link setup

### DISCUSSION THREAD

**Discussion by MississippiDev** [chuot/rdio-scanner#269] [Q&A]
Question 1:
Hey, trying to set up my rdio scanner with the website and URL, but the manual is not showing me how to do it.
I want to know can you help me set it up or tell how me how can I do it?

Question 2:
Are you doing any public safety discounts with the rdio scanner? Because I am a Volunteer Firefighter/EMA in Mississippi.

**Comment by chuot** [comment]
A1: Please be more specific about the configuration you have. Are you running your own instance of Rdio Scanner open source or trying to connect to someone else?

A2: You can request a reimbursement from your first responder employer, as some agree to do.

**Comment by TPSSCW** [comment]
Mississippidev did you ever get this figured out...I'm in the same boat! Instructions are very vague.  I'm close to asking for a fefund on the app :)
